cmd/podman/rmi.go

```diff
@@ -77,3 +98,217 @@ func rmiCmd(c *cli.Context) error {
 	}
 	return lastError
 }
+
+func rmiBuildahCmd(c *cli.Context, imageID string) error {
+
+	store, err := getStore(c)
+	if err != nil {
+		return err
+	}
+	image, err := getImage(imageID, store)
+	if err != nil || image == nil {
+		errors.Wrapf(err, "could not get image %q", imageID)
+	}
+
+	ctrIDs, err := runningContainers(image, store)
+	if err != nil {
+		return errors.Wrapf(err, "error getting running containers for image %q", imageID)
+	}
+	if len(ctrIDs) > 0 && len(image.Names) <= 1 {
```

Comment:
We don't need this logic, the standard logic in our image library for removing images should work fine